src/ldp/http/BasicResponseWriter.ts

```diff
@@ -34,6 +34,10 @@ export class BasicResponseWriter extends ResponseWriter {
 
     if (input.result.data) {
       input.result.data.pipe(input.response);
+      input.result.data.on('error', (error): void => {
+        this.logger.error(`Writing to HttpResponse failed with message ${error.message}`);
```

Comment:
Interesting how TypeScript lets you get away with assuming `error instanceof Error` here, whereas nothing enforces that.

Reply:
Not sure, but I think the typings here are designed intelligently so that the compiler can infer this based on the 'error' event label.

Reply:
Yea, these are the typings:
```typescript
emit(event: "error", err: Error): boolean;
on(event: "error", listener: (err: Error) => void): this;
```